examples/.env.example

```diff
@@ -0,0 +1,3 @@
+IMX_APP_USER_WALLET_PRIVATE_KEY=
+IMX_APP_USER_STARK_PRIVATE_KEY=
+IMX_APP_ALCHEMY_API_KEY=
```

Comment:
Is there a reason you're adding `IMX_APP` as the prefix to all these?

Reply:
Good point. It's a relic of a past project. I'll remove the prefix.

Reply:
[b2a9ab9](https://github.com/immutable/imx-core-sdk/pull/195/commits/b2a9ab9e2a9ebbeefea8344b89fbe6807e754263)